.github/workflows/pr-cleanup.yaml

```diff
@@ -0,0 +1,29 @@
+name: Clean up npm pr tags on release
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  release-pr:
+    name: "Clean up npm pr tags on release"
+    runs-on: ubuntu-latest
+    # comment out if:false to enable release PR to npm
+    if: false
+    permissions: write-all
+    steps:
+      - name: CI Setup
+        uses: ./.github/actions/ci-setup
+
+      - name: Ensure NPM access
+        run: npm whoami
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+      - name: Unpublish the package with the tag @pr-${{ github.event.pull_request.number }} from npm
+        id: unpublish
+        run: |
+          echo ::set-output name=version::$(npm --force unpublish pr-${{ github.event.pull_request.number }})
```

Comment:
I thought we could not access `github.event.pull_request.number` when a workflow is triggered by a push on the `master` branch. Did I get it wrong?

> cc @Torres-ssf

Reply:
Yes @arboleya, I have faced an error because of this while trying to get the proper workflow settings to deploy our new docs to GitHub pages. 

It seems the `github.event.pull_request` property is only available when:
```yaml
name: Pull Request Workflow

on:
  pull_request:
    branches:
      - branchName
```